server/src/org/apache/cloudstack/network/lb/CertServiceImpl.java

```diff
@@ -111,37 +116,37 @@ public CertServiceImpl() {
     @DB
     @Override
     @ActionEvent(eventType = EventTypes.EVENT_LB_CERT_UPLOAD, eventDescription = "Uploading a certificate to cloudstack", async = false)
-    public SslCertResponse uploadSslCert(UploadSslCertCmd certCmd) {
+    public SslCertResponse uploadSslCert(final UploadSslCertCmd certCmd) {
         try {
-            String cert = certCmd.getCert();
-            String key = certCmd.getKey();
-            String password = certCmd.getPassword();
-            String chain = certCmd.getChain();
+            final String cert = certCmd.getCert();
+            final String key = certCmd.getKey();
+            final String password = certCmd.getPassword();
+            final String chain = certCmd.getChain();
 
             validate(cert, key, password, chain);
             s_logger.debug("Certificate Validation succeeded");
 
-            String fingerPrint = generateFingerPrint(parseCertificate(cert));
+            final String fingerPrint = generateFingerPrint(parseCertificate(cert));
 
-            CallContext ctx = CallContext.current();
-            Account caller = ctx.getCallingAccount();
+            final CallContext ctx = CallContext.current();
+            final Account caller = ctx.getCallingAccount();
 
             Account owner = null;
-            if ((certCmd.getAccountName() != null && certCmd.getDomainId() != null) || certCmd.getProjectId() != null) {
+            if (certCmd.getAccountName() != null && certCmd.getDomainId() != null || certCmd.getProjectId() != null) {
                 owner = _accountMgr.finalizeOwner(caller, certCmd.getAccountName(), certCmd.getDomainId(), certCmd.getProjectId());
             } else {
                 owner = caller;
             }
 
-            Long accountId = owner.getId();
-            Long domainId = owner.getDomainId();
+            final Long accountId = owner.getId();
+            final Long domainId = owner.getDomainId();
 
-            SslCertVO certVO = new SslCertVO(cert, key, password, chain, accountId, domainId, fingerPrint);
+            final SslCertVO certVO = new SslCertVO(cert, key, password, chain, accountId, domainId, fingerPrint);
             _sslCertDao.persist(certVO);
 
             return createCertResponse(certVO, null);
 
-        } catch (Exception e) {
+        } catch (final Exception e) {
```

Comment:
I'm not sure, the PR changes are around fixing CloudStack to work with the newer bountycastle dependency and not codebase wide issuse.